src/ontology/hp.Makefile

```diff
@@ -334,4 +334,19 @@ PATTERN_calcifiedAnatomicalEntity="https://docs.google.com/spreadsheets/d/e/2PAC
 PATTERN_calcifiedAnatomicalEntityWithPattern="https://docs.google.com/spreadsheets/d/e/2PACX-1vSFB67ABDEUTWz-O2px0AdFFcNLEm5DlhKp5_haV3M1F2tgG-VcCHKe67qCOe1vKKa-NAWI9icJCQuO/pub?gid=1937862719&single=true&output=tsv"
 calcified:
 	wget $(PATTERN_calcifiedAnatomicalEntity) -O ../patterns/data/default/calcifiedAnatomicalEntity.tsv
-	wget $(PATTERN_calcifiedAnatomicalEntityWithPattern) -O ../patterns/data/default/calcifiedAnatomicalEntityWithCalcificationPattern.tsv
\ No newline at end of file
+	wget $(PATTERN_calcifiedAnatomicalEntityWithPattern) -O ../patterns/data/default/calcifiedAnatomicalEntityWithCalcificationPattern.tsv
+	
+
+HPOA_DIR=hpoa
+
+hpoa:
+	#rm -rf $(TMPDIR)/hpo-annotation-data
+	#rm -rf $(HPOA_DIR) && mkdir $(HPOA_DIR)
+	#cd $(TMPDIR) && git clone https://github.com/monarch-initiative/hpo-annotation-data.git
+	cd $(TMPDIR)/hpo-annotation-data/rare-diseases/ && $(MAKE) -C misc && $(MAKE) -C current
+	sed -i '/HP:0032468/d' ./tmp/hpo-annotation-data/rare-diseases/misc/phenotype_annotation.tab
```

Comment:
- [ ] Do not merge this PR until this issue is addressed. @drseb is working on it.